Source/WebCore/page/PageDebuggable.cpp

```diff
@@ -46,9 +46,6 @@ PageDebuggable::PageDebuggable(Page& page)
 
 String PageDebuggable::name() const
 {
-    if (!m_nameOverride.isNull())
-        return m_nameOverride;
```

Comment:
why was this not enough?  is there a situation where we need _both_ the `name` and `nameOverride`?